lib/network/error.js

```diff
@@ -2,11 +2,10 @@ class TelegramError extends Error {
   constructor (payload = {}) {
     super(`${payload.error_code}: ${payload.description}`)
     this.code = payload.error_code
-    this.response = payload.response
+    this.response = payload
     this.description = payload.description
-    this.retryAfter = payload.retry_after
```

Comment:
Could you also fix `retryAfter` usage here: https://github.com/telegraf/telegraf/blob/develop/lib/telegraf.js#L122 ?